docs/docs/testing-css-in-js.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Testing CSS-in-JS"
+---
+
+Popular CSS-in-JS libraries like [styled-components](https://github.com/styled-components/styled-components) or [emotion](https://github.com/emotion-js/emotion) can also be tested with the help of [jest-styled-components](https://github.com/styled-components/jest-styled-components) or [jest-emotion](https://github.com/emotion-js/emotion/tree/master/packages/jest-emotion) respectively. These packages improve Jest's built-in snapshot testing experience and are a great way to help avoid unintended changes to your website's UI. Please refer to your package's documentation to see if it also offers testing capabilities.
+
+By default snapshots show the generated class names (which we didn't set) and no styling information. When changing the styles you'll only see the diff of some cryptic class names. That's why we should use the above mentioned *snapshotSerializers*.
```

Comment:
Another thought - _snapshotSerializers_ are not directly mentioned earlier. Maybe there could be a paragraph before this that explains what a _snapshotSerialiser_ is, and how `jest-styled-components` or `jest-emotion` use them to improve the testing experience?

Reply:
Yeah, good idea!